framework/include/utils/NestedSolve.h

```diff
@@ -203,7 +220,7 @@ NestedSolveTempl<is_ad>::nonlinear(NestedSolveTempl<is_ad>::DynamicVector & gues
                                    R & computeResidual,
                                    J & computeJacobian)
 {
-  // adaptor functor to utilize the Eigen non-linear solver
+  /// adaptor functor to utilize the Eigen non-linear solver
```

Comment:
these `///` are not utilized by doxygen and should just stay `//`

Reply:
// Resolved